simtools/ray_tracing.py

```diff
@@ -155,7 +155,7 @@ def _initialize_single_mirror_mode(self, mirror_numbers):
         tuple
             Tuple containing source distance and mirror numbers.
         """
-        self._logger.debug(
+        self._logger.info(
```

Comment:
Also think this should be on the debug level, as this is a comment to be used for developers (we have an open issue on it).